### DISCUSSION TITLE
PhoneAuthentication with TextLocal

### DISCUSSION THREAD

**Discussion by vinaypanduga** [supabase/supabase#27230] [Questions]
Hi team
I am getting this error while trying to login with OTP with textlocal 
POST https://aeakbcdznktpsbrfsgys.supabase.co/auth/v1/otp 422 (Unprocessable Content)
and this AuthApiError: Error sending confirmation sms: textlocal error: Invalid template (code: 80) template message: 386070 is the OTP for your Chimple App.\n
Happy Learning.

could anyone help me out

**Comment by nadavl** [comment]
Same here with TextLocal
